.ci/infra/terraform/main.tf

```diff
@@ -381,6 +382,13 @@ resource "google_project_iam_member" "aiplatform_agent_encrypter_decrypter" {
   member  = "serviceAccount:service-${google_project.proj.number}@gcp-sa-aiplatform.iam.gserviceaccount.com"
 }
 
+# TestAccCertificateManagerPublicCAExternalAccountKey_certificateManagerPublicCaExternalAccountKeyExample
+resource "google_project_iam_member" "sa_public_ca_ext_account_key_creator" {
```

Comment:
This appears to be granted by `Owner`, so we can omit it.

Reply:
good call, removed it!